Cython/Compiler/Visitor.py

```diff
@@ -834,7 +834,7 @@ def repr_of(self, node):
                 result += "(name=\"%s\")" % node.name
             elif isinstance(node, ExprNodes.AttributeNode):
                 result += "(type=%s, attribute=\"%s\")" % (repr(node.type), node.attribute)
-            elif isinstance(node, ExprNodes.ConstNode):
+            elif isinstance(node, (ExprNodes.ConstNode, ExprNodes.StringNode)):
```

Comment:
I also snuck in this slight inprovement to `PrintTree`

Reply:
What about using `PyConstNode` instead? And maybe using `%r` for the value instead of `"%s"`?